### PR TITLE
Add almond milk, soy milk, and a method to produce almond milk.

### PR DESCRIPTION
**Pull request by crizengrim**
#### Summary

```SUMMARY: Content "Add almond milk, soy milk, and a method to produce almond milk using shelled almonds and water."```

#### Purpose of change
Almond milk and soy milks are increasingly more commonly found in households around the world. Used for various reasons, more notably, lactose intolerance. 

#### Describe the solution
Seeing as you can be afflicted with lactose intolerance, why not have the chance to come across a few gallon jugs of these dairy-free milks? Or alternatively, produce almond milk with the shelled almonds one often comes across.

#### Describe alternatives you've considered
I've thought about other plant milk alternatives like oat milk, hemp milk, etc. but those are much less common than these two largely popular kinds. (almond and soy)

First PR! ⁽ ´◡`⁾

### CODE REVIEW

**Comment by TheMurderUnicorn** [comment]
The drinks.json has some format issues.  I don't know how to show them in my comment, but the {'s and }'s need to be lined up with the other ones, if that makes sense.  

In the future, if you put your code in http://dev.narc.ro/cataclysm/format.html (with a [ at the start and a ] at the end), it will format your code for you.

**Comment by mlangsdorf** [comment]
For future PRs, there's not need to put [ ] around the PR type in the summary line.  It should be `SUMMARY: Content "Add almond milk, soy milk, and a method to produce almond milk using shelled almonds and water."` not `SUMMARY: [Content] "Add almond milk, soy milk, and a method to produce almond milk using shelled almonds and water."`

You were very close, though, so good job.

**Comment by mlangsdorf** [comment]
jenkins rebuild

**Comment by Photoloss** [comment]
The rag in the recipe probably should not be consumed but rather used as a tool, see e.g. [juice pressing](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/recipes/recipe_food.json#L1475).

**Comment by umsretuza** [comment]
> The rag in the recipe probably should not be consumed but rather used as a tool, see e.g. [juice pressing](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/recipes/recipe_food.json#L1475).

I thought about the rag being used as a tool, but there is the strangeness of having the rag afterward, perfectly usable. Should it be assumed that the character cleaned the rag near the end of the process?

**Comment by Photoloss** [comment]
> Should it be assumed that the character cleaned the rag near the end of the process?

This is what the juice recipes do, since non-drinkable water is not very hard to come by I'd say it's the lesser of two evils over completely destroying the fabric. Aside from everything else you'd be able to use a single rag for bulk-crafting *hundreds* of portions of almond milk without washing it in between.

File: data/json/items/comestibles/drink.json [hunk 1]
```diff
@@ -22,6 +22,48 @@
...
+    "name": "almond milk",
+    "weight": 258,
+    "color": "white",
+    "spoils_in": "3 days",
+    "container": "jug_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "~",
+    "quench": 22,
+    "calories": 50,
+    "description": "Milk some almonds? Not quite, but blend them with water, yes! A dairy-free alternative strong in calcium! Rival to soy milk.",
```
Comment (AMurkin) [inline data/json/items/comestibles/drink.json]:
```suggestion
    "description": "Milk some almonds?  Not quite, but blend them with water, yes!  A dairy-free alternative strong in calcium!  Rival to soy milk.",
```

File: data/json/items/comestibles/drink.json [hunk 2]
```diff
@@ -22,6 +22,48 @@
...
+    "name": "soy milk",
+    "weight": 258,
+    "color": "white",
+    "spoils_in": "3 days",
+    "container": "jug_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "~",
+    "quench": 22,
+    "calories": 110,
+    "description": "Milk some soybeans? Not quite, but blend them with water, yes! A dairy-free alternative strong in protein! Rival to almond milk.",
```
Comment (AMurkin) [inline data/json/items/comestibles/drink.json]:
```suggestion
    "description": "Milk some soybeans?  Not quite, but blend them with water, yes!  A dairy-free alternative strong in protein!  Rival to almond milk.",
```

**Comment by int-ua** [comment]
Didn't FDA banned using "milk" for these products?

**Comment by umsretuza** [comment]
> Didn't FDA banned using "milk" for these products?

Not yet, and they have a long battle ahead of them before they get there. [Especially with a recent court decision that sided firmly with "milk" being an appropriate label.](https://www.barrellalaw.com/wp-content/uploads/2018/03/BlueDiamondBrief.pdf)

**Comment by Rail-Runner** [comment]
Might want to also add a more "primitive" version of recipe that would (I think) use a containing tool and a hammering tool in place of food processor, but taking more time.

**Comment by Ilysen** [comment]
Hey, neat! I think they could do with a small health value, though, like 1 or 2.

**Comment by KorGgenT** [comment]
would you mind citing the nutrition facts? i just want to make sure that they're based on real life values, instead of based on other items in game. i have a hunch that almond milk in particular is probably more of a source of iron, and not vitamin A

**Comment by umsretuza** [comment]
[primary sources for what the nutrition of the milks is based on](https://i.imgur.com/LXG9QBw.jpg) I used others and did a rough average.

**Comment by KorGgenT** [comment]
ok, then calcium should be 11 and iron should be 1. the translation is: ```RDA percentage * .24```

**Comment by umsretuza** [comment]
> ok, then calcium should be 11 and iron should be 1. the translation is: `RDA percentage * .24`

I've decided to use the updated sources for both [soy](https://silk.com/products/original-soymilk) and [almond](https://silk.com/products/original-almondmilk), under the same brand and the same flavor, for consistency.

Thanks for the formula!

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/latest-experimental-features/5582/1106

**Pull request merged**